packages/feedback-service/.env.example

```diff
@@ -0,0 +1,25 @@
+## Uncomment the lines as needed
+
+NAME=local
+NODE_ENV=local
+PRODUCTION=false
+
+## Server URLs
+API=https://localhost:8080
+GRAPHQL_API=https://localhost:8080/playground
+
+## Database
+DB_PATH=localhost
+DB_NAME=db_name
+# DB_USER=
+# DB_PASSWORD=
+
+## Certificate Configuration
+SSL_KEY=/etc/pki/tls/private/server.key
+SSL_CERT=/etc/pki/tls/certs/server.crt
+
+## Cookie Parser
+COOKIE_PARSER=FEEDBACK_SECRET
+
+## Authorization Token
+# AUTH_TOKEN=
```

Comment:
DO NOT edit this file. This is supposed to be an example. Whatever you need should go in a duplicate of this file called `.env`

Reply:
Okay, this file was created automatically when i created directory structure after that i thought there is not use of having this file so deleted this somehow it generated again.
Noted i will not update this.